packages/bootstrap-vue-next/src/components/BAutocomplete/_autocomplete.scss

```
@@ -79,6 +84,68 @@
 
+.b-autocomplete-input-wrapper {
+  position: relative;
+  flex: 1 1 auto;
+  display: flex;
+  align-items: stretch;
+  min-width: 0;
+
+  .form-control {
+    flex: 1 1 auto;
+    min-width: 0;
+    // Remove border adjustments that input-group usually applies; we inherit them from context
```

Stylelint (17.10.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)